tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -8874,6 +8874,11 @@ SpirvEmitter::processIntrinsicInterlockedMethod(const CallExpr *expr,
                                        const CallExpr *callExpr,
                                        uint32_t outputArgIndex) {
     const auto outputArg = callExpr->getArg(outputArgIndex);
+    if (dyn_cast<DeclRefExpr>(outputArg) == nullptr) {
+      emitError("InterlockedCompareExchange requires a reference as output parameter.", outputArg->getExprLoc());
```

Comment:
nit: I think most of our error messages don't end in a period

Reply:
fixed, removed the period.